### DISCUSSION TITLE
How to display svg

### DISCUSSION THREAD

**Discussion by Gavin-Williams** [dotnet/maui#22272] [Q&A]
I've dropped an svg 'wallet.svg' into the /Resources/Images folder of the razor-maui-hybrid template. Then I have referenced it like so...

`<img src="/resources/images/wallet.svg" alt="WalletIcon" width="32" height="32">`

I'm just getting a broken image. What's going on?

**Comment by mhrastegari** [comment]
In the Blazor (Any kind) you need to put your resources like Image, font, JS and main CSS files to the `wwwroot` folder and simply just type the file name.

**Reply by mhrastegari** [reply]
The `Resources` folder is mostly for native (not Blazor and web) stuff like splash screen or app icon or Xaml pages.
